### ISSUE TITLE
Problem with stem colors after a transparent rendering.

### ISSUE THREAD

**Issue by AlbertHart** [opensheetmusicdisplay/opensheetmusicdisplay#614]
In the #1 image below I am rendering a score with transparent notes and stems. (To make the problem easier to see I have made them red, rather than transparent in this image.)

In all three renderings, I am using the same code to set options.

```
osmd.setOptions({
   
        defaultColorNotehead: drawing_color,
        defaultColorStem: drawing_color,
        defaultColorRest: rest_color,
        
        coloringEnabled: coloring_enabled,
        colorStemsLikeNoteheads: 1,
        defaultFontFamily: "Ariel Bold",

        drawFromMeasureNumber: start_measure,
        drawUpToMeasureNumber: end_measure + 1,

    });
```

For #1, drawing_color is set to #ff0000 and coloring enabled is set to 1:

```
       drawing_color: #ff0000,
        rest_color: #ff0000
        coloring_enabled: 1
        start_measure: 1
        end_measure: 36

```

For #2 and #3 I am setting these values:

```
drawing_color: #000000,
        rest_color: #00000000
        coloring_enabled: 0
        start_measure: 1
        end_measure: 3
```

(Ignore the transparent rest color, it is not used because coloring_enabled is set to 0)

In all three cases colorStemsLikeNoteheads is set to 1, but it doesn't work in the #2 rendering, until it is re-rendered in #3

All rendering are made by pressing a button on the screen, and with their own new osmd. So this is probably not a timing issue, but rather an issue of the stem colors set with colorins_enabled in #1 carrying over into #2, but then being set properly when it is re-rendered with #3. 

![image](https://user-images.githubusercontent.com/8965352/68180484-28981480-ff51-11e9-866c-864986d9b2d4.png)

**Comment by AlbertHart** [comment]
The image didn't load above so I am reloading it here.

![image](https://user-images.githubusercontent.com/8965352/68180059-7e6bbd00-ff4f-11e9-8a3f-f3a084a45e23.png)

**Comment by sschmidTU** [comment]
I think i fixed the issue by respecting EngravingRules.ColoringEnabled for stem coloring as well.
Could you test this? Unfortunately i don't have a test setup with 3 osmd renders on one page and coloring.

**Comment by AlbertHart** [comment]
Where do I download the fix?

**Comment by sschmidTU** [comment]
I commited it to develop. I assumed you use the develop branch for your development anyways, or are you using the npm module?
In that case, we'll probably do a new npm release very soon anyways.

**Comment by AlbertHart** [comment]
I do not use the develop branch, and I do not currently build from the source.

I just recently converted to a MAC and am using Visual Code.

Can you point me to instructions on how to use OSMD github on my MAC?

**Comment by AlbertHart** [comment]
I am currently following the instructions at: 
https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/wiki/Build-Instructions

I'll see what happens

**Comment by AlbertHart** [comment]
I got as far as, if you can offer any suggestions.

**npm install**
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN osmd@1.0.0 No description
npm WARN osmd@1.0.0 No repository field.

up to date in 0.395s
found 0 vulnerabilities

Alberts-MacBook-Pro:OSMD alhart$ **ls**
opensheetmusicdisplay	package-lock.json	package.json

Alberts-MacBook-Pro:OSMD alhart$ **npm run build:webpack-dev**
npm ERR! missing script: build:webpack-dev

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alhart/.npm/_logs/2019-11-08T16_01_20_607Z-debug.log

**Comment by sschmidTU** [comment]
Ah, you're in the wrong folder. You need to go to the opensheetmusicdisplay folder, that's the main project folder. there you can run npm install and so on.

**Comment by AlbertHart** [comment]
I was able to build in the proper folder.

However, I cloned:

git clone https://github.com/opensheetmusicdisplay/opensheetmusicdisplay.git

Do I need to clone, or get, something else to get your changes? 

I start seareching the web for "osmd develop branch"

**Comment by bneumann** [comment]
Not sure what the main branch is but you can change it with ```git checkout develop```

**Comment by AlbertHart** [comment]
Would the path or final command be related to:

git clone https://github.com/opensheetmusicdisplay/opensheetmusicdisplay.git

e.g. 

git checkout develop  clone https://github.com/opensheetmusicdisplay/opensheetmusicdisplay.git

**Comment by bneumann** [comment]
Oh sorry I overlooked that. I think it should be ```git clone https://github.com/opensheetmusicdisplay/opensheetmusicdisplay```

**Comment by AlbertHart** [comment]
That is what I have been doing. By I don't know if that gives be the latests changes.

**Comment by bneumann** [comment]
If you already have it, try ```git pull``` to get the latest changes from the server

**Comment by sschmidTU** [comment]
the proper order should be:
`git clone https://github.com/opensheetmusicdisplay/opensheetmusicdisplay.git`
`git checkout develop`

then, when new changes have been coming after your initial clone:
`git pull`

sorry, i forgot about including the develop branch in the description.

**Comment by AlbertHart** [comment]
I think I fetched and built the develop branch properly, but I still have the same problem.

I created and attached a stand-alone .htm which demonstrates the problem.

After changing the .js path and loading it, click "Do improvisation" and it will create a score with transparent notes.

![image](https://user-images.githubusercontent.com/8965352/68547996-7378b880-03a5-11ea-9f36-584079d4c94a.png)

Then click "Do chord diagrams:" and in redraws exacrtly the same xml, but the stem are transparent.

![image](https://user-images.githubusercontent.com/8965352/68548009-90ad8700-03a5-11ea-91ec-713fd1b8e524.png)

Click "Do chord diagrams" again, and it renders properly.

![image](https://user-images.githubusercontent.com/8965352/68548031-c5b9d980-03a5-11ea-8236-a63f69f29692.png)

All three renderings are made into the same container in this test version.

Here is the .htm file.
[osmd_score_test.htm.zip](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/files/3828747/osmd_score_test.htm.zip)
